bundles/automation/org.eclipse.smarthome.automation.api/NOTICE

```diff
@@ -0,0 +1,19 @@
+This content is produced and maintained by the Eclipse SmartHome project.
+
+* Project home: https://eclipse.org/smarthome/
+
+== Declared Project Licenses
+
+This program and the accompanying materials are made available under the terms
+of the Eclipse Public License 2.0 which is available at
+http://www.eclipse.org/legal/epl-2.0.html.
```

Comment:
Please correct this file and let your script run again so all the occurences are fixed. I have also marked the arche-type in case you script is only changing the bundles.